src/PointwiseFunctions/GeneralRelativity/GeneralizedHarmonic/Ricci.hpp

```diff
@@ -0,0 +1,130 @@
+ *
+ *         =& g^{kl}\left(\partial_{k}\Phi_{(ij)l} -
+ *                        \frac{1}{2}\partial_{k}\Phi_{lij}\right)
+ *            - b^{l} (\Phi_{(ij)l} - \frac{1}{2}\Phi_{lij})\nonumber\\
+ *          & - g^{kl}\left(\partial_{i}\Phi_{(kj)l}
+ *                          - \frac{1}{2}\partial_{i}\Phi_{lkj}\right)
+ *            - \Phi_{i}{}^{kl}\left(\Phi_{(kj)l}
+ *                                   - \frac{1}{2}\Phi_{lkj}
+ *                                   \right)\nonumber\\
+ *          & + g^{km}\left(\Phi_{(kl)m} - \frac{1}{2}\Phi_{mkl}\right)
+ *              g^{ln}\left(\Phi_{(ij)n} - \frac{1}{2}\Phi_{nij}\right)
+ *              \nonumber\\
+ *
+ *          & - g^{km}\left(\Phi_{(il)m} - \frac{1}{2}\Phi_{mil}\right)
+ *              g^{ln}\left(\Phi_{(jk)n} - \frac{1}{2}\Phi_{njk}\right).
+ * \f}
+ * Gathering all terms with second derivatives:
+ * \f{align}
+ * R_{i j} =& \frac{1}{2} g^{k l} \left(\partial_k\Phi_{ijl}
+ *                                      + \partial_k\Phi_{jil}
+ *                                      - \partial_k\Phi_{lij}
+ *                                      + \partial_i\Phi_{lkj}
+ *                                      - \partial_i\Phi_{kjl}
+ *                                      - \partial_i\Phi_{jkl}\right)
+ *          + \mathcal{O}(\Phi), \nonumber\\
+ *         =& \frac{1}{2} g^{kl} \left(\partial_{(j}\Phi_{lki)}
+ *                                     - \partial_{(j}\Phi_{i)kl}
+ *                                     + \partial_k \Phi_{(ij)l}
+ *                                     - \partial_l \Phi_{kij} \right)
+ *          + \mathcal{O}(\Phi),
+ * \f}
+ * where we use the four-index constraint
+ * \f$C_{klij}=\partial_k\Phi_{lij}=\partial_l\Phi_{kij}=0\f$ to swap the
+ * first and second derivatives of the spatial metric, and symmetrize
+ * \f$R_{ij} = R_{(ij)}\f$. Similarly gathering the remaining terms and
+ * using the four-index constraint we get:
+ * \f{align}
+ * R_{i j} =& -\frac{1}{2} b^k\left(\Phi_{ijk} + \Phi_{jik} - \Phi_{kij}\right)
+ *            -\frac{1}{2} \Phi_i{}^{kl} \left(\Phi_{jkl} + \Phi_{kjl}
+ *                                            - \Phi_{lkj}\right)\nonumber\\
+ *
+ *         &+ \frac{1}{4} d^k \left(\Phi_{ijk} + \Phi_{jik} - \Phi_{kij}\right)
+ *         - \left(\Phi_{(il)}{}^k - \frac{1}{2} \Phi^k{}_{il}\right)
+ *           \left(\Phi_{(kj)}{}^l - \frac{1}{2} \Phi^l{}_{kj}\right)
+ *         + \mathcal{O}(\partial\Phi) \\
+ *
+ *         =& \frac{1}{4} \left(\Phi_{ijk} + \Phi_{jik} - \Phi_{kij}\right)
+ *            (d^k - 2 b^k)
+ *         + \frac{1}{4} \Phi_{ik}{}^l \Phi_{jl}{}^k
+ *         + \frac{1}{2} \left(\Phi^k{}_{il} \Phi_{kj}{}^l
+ *                             - \Phi^k{}_{li} \Phi^l{}_{kj}\right)
+ *         + \mathcal{O}(\partial\Phi).
+ * \f}
+ * Gathering everyhing together, we compute the spatial Ricci tensor as:
+ * \f{align}
+ * R_{i j} =& \frac{1}{2} g^{kl} \left(\partial_{(j}\Phi_{lki)}
```

Comment:
The symmetry parens first term in the inner sum is confusing -- as it's written it appears to imply a symmetrization over all four indices, where I think you mean just over the j and i indices. The notation I've seen for this case of partial symmetrization is to make the index on the derivative `(j|` and the last index on the \Phi be `|i)`.

Reply:
agreed, I've changed to the piped notation.